src/main.c

```diff
@@ -213,17 +213,26 @@ void c_main(void)
 	 * miss and therefore cause no evictions.
 	 */
 
+	int id = MMC_ID;
 	mmc_inited = !mmc_init(MMC_ID);
+	char *rootfs_dev = "root=/dev/mmcblk0p1";
```

Comment:
```"root=/dev/mmcblk" #MMC_ID "p1"```;

Reply:
I think about it, but I have mmcblk0 as emmc and mmcblk1 as usd, not mmcblk2.